ironfish/src/network/peerNetwork.ts

```diff
@@ -702,8 +702,9 @@ export class PeerNetwork {
       return true
     }
 
+    await this.node.accounts.syncTransaction(verifiedTransaction, {})
```

Comment:
I'm not 100% sure if this is the right order or not, so it might be good to get an explanation on this over a call. I'll just post a bullet point of my thoughts.

 - In all other places syncTransaction is used, pre validation occurs. Since were not doing validation anymore, does this mean someone can craft invalid transactions and poison our wallet? It seems like we would want to do validation.
 -  We are still using the mem pool to gate the TX here, is that fine? If were not synced, and if the mem pool already has it.